### PR TITLE
Make mapping_key be a hash instead of the concatenation of attributes

### PR DESCRIPTION
**Pull request by mauroz77**
- Includes removal of getters/setters in MappingEntity because it's now using Lombok.

### CODE REVIEW

File: data-services/src/main/java/org/pdxfinder/MappingService.java [hunk 1]
```diff
@@ -835,5 +838,10 @@ private void saveEntitiesInContainerToDatabase() {
     log.info("Database rebuild. Mapping data count: " + mappingEntityRepository.count());
   }
 
+  public List<MappingEntity> getAllMappingEntities() {
```
Comment (Afollet) [inline data-services/src/main/java/org/pdxfinder/MappingService.java:841]: This is going to be used on an API?

File: data-model/src/main/java/org/pdxfinder/MappingEntity.java [hunk 2]
```diff
@@ -10,12 +10,15 @@
 import java.util.stream.Collectors;
 import lombok.EqualsAndHashCode;
 import lombok.Getter;
+import lombok.RequiredArgsConstructor;
```
Comment (Afollet) [inline data-model/src/main/java/org/pdxfinder/MappingEntity.java:13]: lombok, nice!

**Review by Afollet — COMMENTED** [review]
Added commit to remove unused import

File: data-services/src/main/java/org/pdxfinder/MappingService.java (hunk 1 above)
Comment (mauroz77) [reply data-services/src/main/java/org/pdxfinder/MappingService.java:841]: No for now, I'll remove it

**Pull request merged**